lib/ansible/parsing/vault/__init__.py

```diff
@@ -75,7 +74,7 @@
 from ansible.errors import AnsibleError
 from ansible import constants as C
 from ansible.module_utils.six import PY3, binary_type
-from ansible.module_utils.six.moves import zip
+from ansible.module_utils.six.moves import zip as _zip
```

Comment:
I just checked and six.moves.zip always comes from Python.  In Python2, it's in builtins (and thus available globally) whereas in python3 it's from itertools.izip.

If you can figure out the invocation to add ansible.module_utils.six.moves that sounds good.  There's currently not an ansible pylint config file but you can modify the command line invocation of pylint here: https://github.com/ansible/ansible/blob/devel/test/runner/lib/sanity.py#L466